### PR TITLE
Add PointList sampling for spectrogram, superceding SinglePoint sampling

### PR DESCRIPTION
**Pull request by dbruneau-mie**
Add PointList sampling for spectrograms
This allows us to make multi-point sonifications from list of points
Add "all" option for spectrogram components, superceding the need for "create_spectrogram_from_components.py"
This also allows us to make sonifications including all components

### CODE REVIEW

File: src/vasp/automatedPostprocessing/postprocessing_h5py/spectrograms.py [hunk 1]
```diff
@@ -77,13 +77,15 @@ def read_command_line_spec() -> configargparse.Namespace:
                         help="Component of the data to visualize. Choose 'x', 'y', 'z', or 'mag' (magnitude).")
```
Comment (keiyamamo) [inline src/vasp/automatedPostprocessing/postprocessing_h5py/spectrograms.py:77]: Should we add `all` as an option inside help?

File: src/vasp/automatedPostprocessing/postprocessing_h5py/spectrograms.py [hunk 2]
```diff
@@ -77,13 +77,15 @@ def read_command_line_spec() -> configargparse.Namespace:
...
     parser.add_argument('--sampling-method', type=str, default="RandomPoint",
                         help="Sampling method for spectrogram generation. Choose from 'RandomPoint' (random nodes), "
-                             "'SinglePoint' (single point specified by '--point-id'), or 'Spatial' (ensures uniform "
+                             "'PointList' (single point specified by '--point-id'), or 'Spatial' (ensures uniform "
```
Comment (keiyamamo) [inline src/vasp/automatedPostprocessing/postprocessing_h5py/spectrograms.py:80]: should be `--point-ids` instead of `--point-id`?

**Review by keiyamamo — APPROVED** [review]
@dbruneau-mie  Looks great! Just some minor comments. When this gets merged, can we remove `create_spectrogram_from_components.py`?

**Comment by dbruneau-mie** [comment]
> @dbruneau-mie Looks great! Just some minor comments. When this gets merged, can we remove `create_spectrogram_from_components.py`?

yes we can remove that script.

**Pull request merged**